benchmarks/cpp/nvfuser/utils.cpp

```diff
@@ -142,20 +142,28 @@ void runBenchmarkIterations(
     std::vector<c10::IValue>& aten_inputs) {
   fusion_executor_cache->runFusionWithInputs(aten_inputs);
   bool segmented =
-      fusion_executor_cache->getMostRecentKernelRuntime()->isSegmented();
```

Comment:
Adding a note to myself to unify the two possible paths to get a practically unsegmented fusion.

Reply:
Yeah, this might not be a big deal in the future as we move away from welford translation into the individual reductions. My previous look at this perf-wise indicated using welford was generally faster.